### PR TITLE
RHODS-12779: Adding a non-breaking space to 'Red Hat'

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Added the org-name attribute to use in place of 'Red Hat'. The new attribute includes a non-breaking space, per style guidance.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: assemblies/support-requirements-and-limitations.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ ifdef::context[:parent-context: {context}]
 
 [role="_abstract"]
 ifndef::upstream[]
-Review this section to understand the requirements for Red Hat support and any limitations to Red Hat support of Red Hat OpenShift Data Science.
+Review this section to understand the requirements for {org-name} support and any limitations to {org-name} support of {org-name} OpenShift Data Science.
```
Comment (jbyrne-redhat) [inline assemblies/support-requirements-and-limitations.adoc:13]: Can use the {productname-long} attribute in the last instance here.

File: assemblies/working-with-data-science-pipelines.adoc [hunk 2]
```diff
@@ -35,7 +35,7 @@ ifdef::upstream[]
...
 endif::[]
 ifndef::upstream[]
-Before you can use data science pipelines, you must install the OpenShift Pipelines operator. For more information about installing a compatible version of the OpenShift Pipelines operator, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/cicd/pipelines#op-release-notes[Red Hat OpenShift Pipelines release notes] and link:https://access.redhat.com/articles/6986416[Red Hat Openshift Data Science: Supported Configurations].
+Before you can use data science pipelines, you must install the OpenShift Pipelines operator. For more information about installing a compatible version of the OpenShift Pipelines operator, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/cicd/pipelines#op-release-notes[{org-name} OpenShift Pipelines release notes] and link:https://access.redhat.com/articles/6986416[{org-name} Openshift Data Science: Supported Configurations].
```
Comment (jbyrne-redhat) [inline assemblies/working-with-data-science-pipelines.adoc:38]: Again, can use the {productname-long} attribute in the last instance.

File: modules/overview-of-pipelines-in-jupyterlab.adoc [hunk 3]
```diff
@@ -10,7 +10,7 @@ ifdef::upstream[]
...
 endif::[]
 ifndef::upstream[]
-Before you can work with pipelines in JupyterLab, you must install the OpenShift Pipelines operator. For more information about installing a compatible version of the OpenShift Pipelines operator, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/cicd/pipelines#op-release-notes[Red Hat OpenShift Pipelines release notes] and link:https://access.redhat.com/articles/6986416[Red Hat Openshift Data Science: Supported Configurations].
+Before you can work with pipelines in JupyterLab, you must install the OpenShift Pipelines operator. For more information about installing a compatible version of the OpenShift Pipelines operator, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/cicd/pipelines#op-release-notes[{org-name} OpenShift Pipelines release notes] and link:https://access.redhat.com/articles/6986416[{org-name} Openshift Data Science: Supported Configurations].
```
Comment (jbyrne-redhat) [inline modules/overview-of-pipelines-in-jupyterlab.adoc:13]: Same comment as https://github.com/opendatahub-io/opendatahub-documentation/pull/61/files#r1386351190.

**Comment by grainnejenningsRH** [comment]
Thanks @jbyrne-redhat, have made those changes.

**Pull request merged**